programmieren lernen/python/arbeitsblaetter/00 - Python zuhause.tex

```diff
@@ -43,4 +43,4 @@
 	\end{enumerate}
 	
 	
-\end{document}
\ No newline at end of file
+\end{document}
```

Comment:
Oh, I think that's just a newline added by my editor. Files should always end with a newline.